azure-pipelines/build/linux/du/templates/doclient-lite-docker-steps.yml

```diff
@@ -51,3 +82,13 @@ steps:
       client-lite/deliveryoptimization-agent.dbg
     TargetFolder: '$(Build.ArtifactStagingDirectory)/agent-${{parameters.targetOsArch}}-${{parameters.config}}'
   displayName: 'Populate debug symbols to staging dir'
+
+- task: CopyFiles@2
+  condition: eq('${{parameters.skipTests}}', false)
+  inputs:
+    SourceFolder: '/tmp/build-deliveryoptimization-agent-${{parameters.targetOsArch}}/linux-${{parameters.config}}'
+    Contents: |
+      client-lite/test/deliveryoptimization-agent-tests
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/${{parameters.targetOsArch}}-${{parameters.config}}'
```

Comment:
>$ [](http://example.com/codeflow?start=53&length=1)

The TargetFolder here should have the "agent-" prefix like in the above step. Otherwise outputs goes into two separate directories.